src/main/java/com/google/devtools/build/lib/remote/RemoteExecutionService.java

```diff
@@ -585,8 +592,20 @@ public RemoteAction buildRemoteAction(Spawn spawn, SpawnExecutionContext context
         platform = PlatformUtils.getPlatformProto(spawn, remoteOptions);
       }
 
+      var useOutputPaths = true;
```

Comment:
To be on the safe side, can we add a flag so we don't accidentally break people? We can flip the flag in 8 and remove it in 9.